speechbrain/lobes/models/transformer/Transformer.py

```diff
@@ -440,6 +442,8 @@ def __init__(
             ]
         )
         self.norm = sb.nnet.normalization.LayerNorm(d_model, eps=1e-6)
+        self.layerdrop_prob = layerdrop_prob
+        self.rng = np.random.default_rng()
```

Comment:
So we do want this randomness. If the seed were set then the layers dropped will be the same across nodes and that will cause a large drop in performance (notice I do not have a seed in the yaml).

Even if we were to take the rank (node ID) into account while setting the seed, this leads to problems when restarting a build because you are going to be using the same layerdrop and dropout masks as when you started training. Properly setting the seed for DDP training that allows restarting training is not trivial, a hacky solution is possible but I preferred keeping the code clean.

Reply:
I understand, but a huge point with SB is that everything should be replicable. Can we replace numpy with torch.rand so users don't have to install numpy ?